### PR TITLE
New marks mechanism

### PR DESCRIPTION
**Pull request by FrankMittelbach**
*Pull requests in this repository are intended for LaTeX Team members only.*

# Internal housekeeping

## Status of pull request

- Feedback wanted 
- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

This still needs the news entry but otherwise please review

### CODE REVIEW

File: base/doc/ltnews35.tex [hunk 1]
```diff
@@ -218,6 +218,71 @@ \section{The \pkg{latex-lab} bundle}
...
+
+
+The new mechanism now available in \LaTeX{} starting with the 2022
+release overcomes both issues:
+\begin{itemize}
+\item
+  It provides arbitrary many, fully independent named marks, that
+  can be allocated and from that point onwards used.
+\item
+  It offers access for each such marks to to retrieve its, top,
```
Comment (dbitouze) [inline base/doc/ltnews35.tex:269]: "to to" → "to"

Comment (dbitouze) [reply base/doc/ltnews35.tex:269]: "its, top," → "its top,"?

File: base/doc/ltnews35.tex [hunk 2]
```diff
@@ -218,6 +218,71 @@ \section{The \pkg{latex-lab} bundle}
...
+  first and bottom value.
```
Comment (dbitouze) [inline base/doc/ltnews35.tex:270]: "value" → "values"

Comment (FrankMittelbach) [reply base/doc/ltnews35.tex:270]: not sure about that one I think it should be singular, I leave that to copyediting by Karl later.

**Pull request merged**